docs/source/primitives.md

```diff
@@ -15,6 +15,7 @@ $$
 \text{Linear} &:  \mathbb{R}^{m \times n} \\
 \text{Conv1d} &:  \mathbb{R}^{c_{out} \times c_{in} \times x},  \text{Strides}, \text{Groups}, ... \\
 \text{Conv2d} &:  \mathbb{R}^{c_{out} \times c_{in} \times y \times x},  \text{Strides}, \text{Groups}, ... \\
+\text{SumPool2d} &: \matchbb{R}^{c \times y \times x}, \text{Pooling}, \text{Strides}, \text{Padding} ...\\
```

Comment:
Think this should be `\mathbb{R}`, no?

But this should be updated and formatted anyway with the table from the paper, I think.